src/Compilers/CSharp/Portable/Compilation/MethodBodySemanticModel.cs

```diff
@@ -308,5 +308,12 @@ protected override void AnalyzeBoundNodeNullability(BoundNode boundRoot, Binder
         {
             NullableWalker.AnalyzeWithoutRewrite(Compilation, MemberSymbol, boundRoot, binder, diagnostics, createSnapshots);
         }
+
+        protected override bool IsNullableAnalysisEnabled()
```

Comment:
>IsNullableAnalysisEnabled [](start = 32, length = 25)

Is this going to give us expected result for a top level entry point symbol and for record's primary constructor?